src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques6 {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
```

Comment:
The method name 'newMethod' is not descriptive. It's recommended to use a more meaningful name that reflects the purpose of the method.